addons/l10n_kz/demo/demo_company.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_demo_company_kz" model="res.partner">
+        <field name="name">KZ Company</field>
+        <field name="vat">090241007868</field>
+        <field name="street">Respublika Avenue 4</field>
+        <field name="city">Shymkent</field>
+        <field name="country_id" ref="base.kz"/>
+        
+        <field name="zip">160012</field>
+        <field name="phone">+77252535000</field>
+        <field name="email">info@meyramgroup.kz</field>
```

Comment:
we certainly do want want real data, address mails or phone numbers in the demo data

seriously :facepalm: